package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-react-components",
-  "version": "0.12.1",
+  "version": "0.13.0",
```

Comment:
Is this a precaution in case something breaks? (Or a correction?)
Seems like this should be a patch.

Reply:
Yup... the first one. Agreed it _probably_ should be a patch, but as it _may_ break something, not a patch.